fhir-persistence-schema/docs/DropTenant.md

```diff
@@ -0,0 +1,129 @@
+# Removing a tenant from the database
+
+Use this guide when removing a tenant from the Db2 multi-tenant database. The tenant's data will be removed from the database along with any associated storage.
+
+To obtain a list of the tenants currently managed in a given database, run the following:
+
+```sh
+    java -jar schema/fhir-persistence-schema-*-cli.jar \
+      --prop-file db2.properties --list-tenants
+```
+
+```
+ TenantId     Status       TenantName Schema
+        1  ALLOCATED            test1 FHIRDATA
+        2  ALLOCATED            test2 FHIRDATA2
+        3    DROPPED            test3 <not-known>
+```
+
```

Comment:
do we add the schema info to the tanant table now? or we still have to visit the system table as you showed me to get the schema info for a tenant.

Reply:
I join against the syscat.datapartitions table and look for a table there to sniff out the schema.